contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -1012,6 +1017,17 @@ def test_aaf_composition_metadata_mastermob(self):
 
         self._verify_user_comments(aaf_metadata, expected_md)
 
+    def test_aaf_multiple_timecode_objects(self):
+        """
+        Make sure we can read AAFs with multiple timecode objects of the
+        same start value and length.
+        """
+
+        timeline = otio.adapters.read_from_file(
+            MULTIPLE_TIMECODE_OBJECTS_PATH)
```

Comment:
It would be good to check the value of the relevant timecode extracted from the file.

Reply:
I don't think that is too relevant for this test-case, but I'll add the timecode-value-check, does not do any harm.

Reply:
I added some timecode check